src/ros_filter.cpp

```diff
@@ -2401,12 +2401,14 @@ namespace RobotLocalization
     // It's unlikely that we'll get a velocity measurement in another frame, but
     // we have to handle the situation.
     tf2::Transform targetFrameTrans;
+    bool silentchecktf;
+    nh_.getParam("/silentchecktf",silentchecktf);
     bool canTransform = RosFilterUtilities::lookupTransformSafe(tfBuffer_,
                                                                 targetFrame,
                                                                 msgFrame,
                                                                 msg->header.stamp,
                                                                 tfTimeout_,
-                                                                targetFrameTrans);
+                                                                targetFrameTrans,silentchecktf);
```

Comment:
Would you mind following the existing style for the other parameters (line break, put in line with the other params)?

Reply:
i change the files now is that now the correct format?

Reply:
I added some suggestions. Would you mind incorporating them into the PR? And then, once it passes CI, we can merge.